components/automate-ui/src/app/services/chef-session/chef-session.service.ts

```diff
@@ -108,7 +98,8 @@ export class ChefSessionService implements CanActivate {
   private refresh(): Observable<string> {
     if (!this.id_token) {
       this.isRefreshing = false;
-      return throwError('id_token is yet to be retrieved');
+      console.log('id_token is yet to be retrieved');
+      return throwError('error');
```

Comment:
Should we throw the error properly? A string error is not very descriptive.

Reply:
Yes, had added only error just to avoid duplicity in console.log and throwError, will show the same error in both.